.github/workflows/tests.yaml

```diff
@@ -10,39 +10,108 @@ on:
       - dev-*
 
 jobs:
-  build_tensorflow:
+  build_tensorflow_autotools:
     runs-on: ubuntu-18.04
-    container: ridgerun/r2inference:v0.1.1
+    container: ridgerun/r2inference:v0.1.4
     steps:
     - uses: actions/checkout@v2
     - name: Install backend version
       run: |
         tar -C /usr/local -xzf /root/r2inference/backends/tensorflow/v1.15.0/libtensorflow-cpu-linux-x86_64-1.15.0.tar.gz
-        ldconfig     
+        ldconfig
     - name: Configure the project
       run: |
         NOCONFIGURE=1 ./autogen.sh
         ./configure --disable-docs --enable-tensorflow
-    - name: Build
+    - name: Build library
       run: make
     - name: Run tests
       run: make check
-  build_tflite:
+    - name: Install library
+      run: |
+        make install
+    - name: Build example
+      env:
+        LD_LIBRARY_PATH: /usr/local/lib/
+      run: |
+        cd examples/external
+        make
+        ./list-backends
```

Comment:
If this fails, does the CI trigger and notify the error?

Reply:
What do you mean by notifying the error? I'm pretty sure that if this step fails then that whole job will not pass. It is possibly to see the errors logs afterwards.

Reply:
Understood